batch-setup/cron.py

```diff
@@ -505,9 +505,16 @@ def is_power_of_two(x):
                         'tileops software to use on the TPS instance.')
     parser.add_argument('--metatile-size', default=8, type=int,
                         help='Metatile size (in 256px tiles).')
+    parser.add_argument('--meta-date-prefix', help='Meta tile bucket '
+                        'date prefix, defaults to planet date. You can '
+                        'also set the environment variable '
+                        'META_DATE_PREFIX.')
 
     args = parser.parse_args()
     planet_date = datetime.strptime(args.date, '%y%m%d')
+    meta_date_prefix = (args.meta_date_prefix or
+                        os.environ.get('META_DATE_PREFIX') or
+                        planet_date)
```

Comment:
`datetime.strptime()` returns a `datetime.datetime` object and not a string, which might not be what you expected as all these other things are strings.

Reply:
Good catch! Yes, that wasn't what I was expecting, and it's because I used `planet_date` as the fallback value when I _meant_ to use `date_prefix`. Fixed in 1b113d9.

Thanks!